scripts/download-ffmpeg.js

```diff
@@ -9,7 +9,7 @@ const ora = require('ora');
 
 let spinner = ora({text: 'Installing 7zip', stream: process.stdout}).start();
 
-const FFMPEG_URL = 'https://evermeet.cx/ffmpeg/ffmpeg-85767-gdec2fa8.7z';
+const FFMPEG_URL = 'https://evermeet.cx/pub/ffmpeg/snapshots/ffmpeg-86910-g797c232.7z';
```

Comment:
This is causing a merge conflict, and before I resolve it I just want to make sure this is okay.  I was getting a 404 on `npm install` before making this change.